### PR TITLE
added Cisco Small Business doc

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Added Cisco Small Business switches documentation

# Impacts
Doc, switch module was already there

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -2013,6 +2013,77 @@ On each interface:
   switchport mode access 
   switchport access vlan 4 
 
+Cisco Small Business
+~~~~~~~~~~~~~~~~~~~~
+
+The Cisco Small Business switches support MAC based authentication (MAB) as well as 802.1X and VoIP. All technologies can be combined and will work correctly together on the same switchport.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: All -> Both

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 2]
```diff
@@ -2013,6 +2013,77 @@ On each interface:
...
+
+Nothing is required to activate VoIP on the switch, you must simply configure the voice VLAN you want PacketFence to assign in the PacketFence switch configuration as well as enabling VoIP there as well. Note that your phones must *not* tag their packets on the network and should send their traffic untagged when connected into a PacketFence enabled port. This means you should not have the voice VLAN capabilities enabled on the switch itself as they might conflict with the authorization attributes returned by PacketFence.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: as well as enabling VoIP there as well -> as well as enabling VoIP there.

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 3]
```diff
@@ -2013,6 +2013,77 @@ On each interface:
...
+   dot1x timeout server-timeout 5 
+   dot1x timeout supp-timeout 3 
+   dot1x authentication dot1x mac 
+   dot1x radius-attributes vlan 
+   dot1x port-control auto 
+   spanning-tree portfast 
+   switchport mode general 
+   switchport general pvid 2 
+
+Once you have configured your switchports, you must configure the switch in PacketFence with the following parameters.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: .. parameters: ?

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 4]
```diff
@@ -2013,6 +2013,77 @@ On each interface:
...
+
+  aaa accounting dot1x start-stop group radius
+
+  snmp-server community public ro view Default 
+  snmp-server community private rw view Default 
+
+MAC Authentication
+++++++++++++++++++
+
+In order to configure MAC Authentication, you must enable it on each interface
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Authentication -> authentication ?

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 5]
```diff
@@ -2013,6 +2013,77 @@ On each interface:
...
+   dot1x radius-attributes vlan 
+   dot1x port-control auto 
+   spanning-tree portfast 
+   switchport mode general 
+   switchport general pvid 2 
+
+802.1X with MAB
++++++++++++++++
+
+In order to configure 802.1X with a fall-back to MAC Authentication, you must enable it on each interface
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Authentication -> authentication ?

**Comment by julsemaan** [comment]
fixaide

**Pull request merged**